.circleci/README.md

```diff
@@ -0,0 +1,24 @@
+## CircleCI Project Maintenance Guideline
+
+### 1. Environment Variables
+
+Set environment variables on CircleCi by following [this guide](https://circleci.com/blog/new-on-circleci-import-project-environment-variables/).
+The CircleCi environment variables used in this project are:
+
+In the [Context](https://circleci.com/gh/organizations/scalar-labs/settings#contexts). Current context name is `scalar`.
+<pre>
+    <b>$DOCKERHUB_USERNAME</b>
+    <b>$DOCKERHUB_PASSWORD</b>
+</pre>
+In the [Environment Variables](https://circleci.com/blog/new-on-circleci-import-project-environment-variables/)
+<pre>
+    <b>$SCHEMA_CONTENT</b>
+</pre>
+
+### 2. Trigger CircleCi integration test
```

Comment:
It's not consistent with `1. Environment Variables` since it starts with noun (and V is capital) and this starts with verb (and integration/test are not capital.)
So, please make it something like this.

```suggestion
### 2. Integration Tests
```